config.json

```diff
@@ -1,160 +1,44 @@
 {
   "source": ["data/tokens.json"],
+
   "platforms": {
-    "scss": {
+    "Owlery": {
```

Comment:
I like how you're specifying which library it's for

Reply:
yeah i thought it kept it clean! obvi we can change it around to however we want if this doesn't work for us moving forward - but easy place to start anyways